analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.PatternMatching.cs

```diff
@@ -287,5 +288,246 @@ public void DeclarationPattern_Var_Deconstruction_DoesNotSetConstraint()
             validator.ValidateTag("D", x => x.Should().BeNull());
             validator.ValidateTag("End", x => x.HasConstraint(TestConstraint.First).Should().BeTrue());
         }
+
+        [DataTestMethod]
+        [DataRow("objectNotNull", "null", false)]
```

Comment:
Let's merge `string variableName, string isPattern` to `string expression`, because this is more clear and we don't need to split them for other reasons:
```suggestion
        [DataRow("objectNotNull is null", false)]
```